backend/core/src/listings/combined/combined-listing.entity.ts

```diff
@@ -0,0 +1,411 @@
+import { UnitsSummarized } from "../../units/types/units-summarized"
+import { ViewColumn, ViewEntity } from "typeorm"
+
+const viewQuery = `(
```

Comment:
After reading the whole CL, I think I see what this is doing?

Given this implementation my impression is that if we wanted to change the response of /listings/combined (e.g. add a new feature), we do that by updating this huge raw SQL blob in two places -- once in ExternalListings and once in Listings. This bypasses a lot of the existing infrastructure and isn't super friendly towards new feature dev but if you could confirm  that'd be great, I'm still not super clear on what this is doing

Reply:
Yes, the ViewQuery expression takes either an SQL string or a QueryBuilder object, and in this case we're doing the former.  TypeORM, like most other ORMs, does not play well with things it doesn't know about.  For example, when I tried to have it generate indices on JSON properties (`jurisdiction->>'id'`, etc), it failed because it didn't recognize that field.  In this case, we're generating JSON objects on the fly, and rather than rewriting the entire query in QueryBuilder and hoping it will work, I just left it in its original form.  I do think it could probably stand to be broken up a bit though.  Do you think that would address your concerns?  Should I try to use QueryBuilder for the parts where it might work and just hardcode the rest?